src/jit/regset.cpp

```diff
@@ -1604,7 +1604,7 @@ void RegSet::rsSpillTree(regNumber reg, GenTreePtr tree, unsigned regIdx /* =0 *
 #endif // _TARGET_ARM_
     else
     {
-        assert(!varTypeIsMultiReg(tree));
+        assert(!varTypeIsMultiReg(tree) || (m_rsCompiler->opts.compUseSoftFP && treeType == TYP_LONG));
```

Comment:
Thank you for your comment. Would that mean `GenTreePutArgReg`(for arm) should have the field `gtSpillFlags` and corresponding setter/getters just like `GenTreeCall` and `GenTreePutArgSplit` do?